src/components/templates/Jazzbar/JazzTab/JazzTab.scss

```diff
@@ -63,20 +65,27 @@
         flex-direction: column;
         align-items: center;
         background-color: rgb(25, 24, 26);
-        width: 80%;
+        width: 90%;
         overflow: hidden;
 
-        padding: 0 10px 10px;
+        padding: 0 10px 5px;
         border-radius: 0px 0px 28px 28px;
         .emoji-container {
           display: flex;
           justify-content: space-around;
-          margin: 20px 0;
+          margin: 10px 0;
           font-size: 27px;
           letter-spacing: 0.5rem;
           overflow-x: auto;
+          overflow-y: hidden;
```

Comment:
Why do we want to hide `overflow-y`? Presumably it would be better if it didn't overflow at all yeah?

Reply:
Sometimes it gets a little squeezed on small-height devices. Not critical, but @Sparkle-T noticed it.

Reply:
Should we make it scroll in that case then instead?

Reply:
It is centered vertically. I guess it was squeezed by a few pixels, which didn't result in any visibility loss. The problem was that an ugly scrollbar appeared here. 

![image](https://user-images.githubusercontent.com/63194656/109814679-7309ce00-7c37-11eb-92a8-3b8e35977a1d.png)

Reply:
Hrmm, fair enough. Probably not a huge issue. Would probably be 'better' to figure/fix the 'few pixels', but it's unlikely to cause big issues given the current layout of it/etc. Not a blocker.